### PR TITLE
creating groups for sops.yaml

### PR DESCRIPTION
**Pull request by RichieCahill**
I used yaml anchors
https://en.wikipedia.org/wiki/YAML#Advanced_components
I also ran the redhat YAML formater

### CODE REVIEW

**Comment by RichieCahill** [comment]
YALL NEED TO VEARIFI THIS. I have but i dont want us to get locked out

**Comment by DerDennisOP** [comment]
not sure if that is possible, could be idk. havn't tested it yet

**Comment by RichieCahill** [comment]
Thank you i didn't know about the help wanted label

**Comment by ahuston-0** [comment]
> YALL NEED TO VEARIFI THIS. I have but i dont want us to get locked out

The passwords aren't currently linked directly to our accounts due to some tomfuckery with `users.mutableUsers` that I mentioned in Discord. So we should be able to test this on jeeves-jr or palatine-hill without blocking us out

**Comment by ahuston-0** [comment]
image
I cant tell if it did none of what we wanted or all of it lol

**Comment by ahuston-0** [comment]
image

y'all i think it fucking worked

**Review by DerDennisOP — APPROVED** [review]
next step would be to hash our user passwords

File: .sops.yaml [hunk 1]
```diff
@@ -10,61 +10,51 @@ keys:
   - &photon age13aqs2mx66dmnyd74sygnwavufv6a4pwnvcqj4ud4l94jk0tjgu6ss57slw
   - &jeeves-jr age1lffr5f5nz0nrenv3ekgy27e8sztsx4gfp3hfymkz77mqaa5a4gts0ncrrh
 
+admins: &admins
+  - *admin_alice
+  - *admin_dennis
+  - *admin_richie
+
+serves: &serves
```
[the remark this answers is not in the document]
Comment (DerDennisOP) [reply .sops.yaml:18]: also this needs fixing asap

[the remark this answers is not in the document]
Comment (RichieCahill) [reply .sops.yaml:18]: y did you approve it with a spelling mistake lol

**Comment by ahuston-0** [comment]
> next step would be to hash our user passwords

I already did that since nix wants the password line as hashed anyway. Will send the command in discord later.

**Comment by ahuston-0** [comment]
I don't think it will let me approve from mobile so approved

**Review by ahuston-0 — APPROVED** [review]
Looks good, found the button

File: .sops.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahuston-0) [reply .sops.yaml]: Sorry this was a draft from last night lol

**Pull request merged**